lib/season_stats.rb

```diff
@@ -31,6 +31,56 @@ def game_teams_by_season(season)
     game_teams
   end
 
+  def total_games_per_coach(season)
+    total = Hash.new(0)
+    game_teams_by_season(season).each do |game_team|
+      total[game_team.head_coach] += 1.0
+    end
+    total
+  end
+
+  def total_wins_per_coach(season)
```

Comment:
These methods look good, lets double check we have all the tests setup for the helpers.